Installing.md

```diff
@@ -78,54 +78,49 @@ Replacing *SERVERADRESS* with the adress/IP of your MySQL server, *USERNAME/PASS
 
 For example it should look something like his:
 
-```mysql -h127.0.0.1 -u root -proot < authserver.sql```
+```mysql -h127.0.0.1 -u root -proot < auth.sql```
 
 That's all you need to do for the database set up.
 ## Part 4 Map and DBC Data
+Now we will extract the Map and DBC data from the client. You will need the WotLK client downloaded for this step.
 
-***TODO As of writing this there is no way to actually set this up by yourself BUT the server also doesn't require either (at the time of writing) so it should be fine anyway. Ignore the step in Part 5 where you are told to copy the map data***
- 
+1. Copy the *Dataextrator.exe* file from *Part 2* to where the *Data* directoy is (It is in the same directory as the *WoW.exe* file)
+2. Run *Dataextractor.exe* and wait for it to finish. There should now be a folder called "NovusExtractor" containing *DBCImportData.sql* and a folder called "maps".
+3. Execute *DBCImportData.sql*. You will need to manually select the database you want to import the data to which changes the command you'd want to execute if doing it from the commandline to the following:
```

Comment:
"select the database you want to import the data to which changes the command you'd want to execture if doing it from the commandline to the following:"

Could this be changed to
"select the database you want to import the data to, this will change the command you execute if you are using the commandline to the following:"
"